### ISSUE TITLE
eslint auto fix doesn't work when vscode auto save is enabled

### ISSUE THREAD

**Issue by youngha-sung** [microsoft/vscode-eslint#1403]
I am trying to trigger the eslint auto fix when file is saved. However when I use vscode's autosave feature, eslint auto fix doesn't get triggered. It's working fine when I don't have autosave enabled and just manually hit cmd + s to save the file. 

OS: OSX
settings.json:
```
{
    "workbench.colorTheme": "Webstorm IntelliJ Darcula Theme",
    "editor.fontSize": 16,
    "editor.formatOnSave": false,
    "editor.codeActionsOnSave": {
        "source.fixAll.eslint": true
    },
    "eslint.alwaysShowStatus": true,
    "files.autoSave": "afterDelay",
    "workbench.editor.enablePreview": false,
    "workbench.iconTheme": "webstorm-icons",
    "editor.unusualLineTerminators": "auto",
    "workbench.settings.editor": "json"
}
```

Is there anything I am missing in the configuration to make this work? or is this a bug that needs to be fixed?

**Comment by dbaeumer** [comment]
This is designed that way in VS Code itself (it is not under control of this extension). When auto save is on `auto fix` is disabled. However you can trigger it manually when in auto save mode by pressing `cmd + s`

**Comment by youngha-sung** [comment]
@dbaeumer Ah, I see. NIT: It would be nice if it's documented from the extension side even though it's controlled from VS Code.

**Comment by juanbomfim22** [comment]
@youngha-sung as a workaround there's this extension [Run on Save](https://github.com/emeraldwalk/vscode-runonsave) that can execute a pre-determined command on specified conditions. 

For instance, I've configured this ` "cmd": "./node_modules/.bin/eslint --fix ${file}"` to run. The save action is triggered by the autosave of VSCode and then it lints the code.
